### ISSUE TITLE
[InputGroup] id attribute ignored when label supplied

### ISSUE THREAD

**Issue by mburtch** [telus/tds-community#347]
## Description
It is expected that when an `id` prop is supplied it would rendered as an HTML id attribute on the input (and htmlFor on the label). However, when both `id` and `label` props are supplied, `id` is ignored and and the id attribute is generated based on `label`.

## Reproduction Steps
-  `<InputGroup label="label" id="id" />`
- Inspect the input, and check the `id` attribute.
- Note that it is based on the label: `<input id="label123" />`

## Workaround details
Exclude the label prop, and create a separate label adjacent to the InputGroup:
```jsx
<Box between={2}>
  <label htmlFor={id}><Text bold>The Label</Text><label>
  <InputGroup id={id} />
</Box>
```

## Recommendation
Correct the condition here to favour the id when supplied:
https://github.com/telus/tds-community/blob/master/packages/InputGroup/InputGroup.jsx#L29

## Meta
- TDS version (npm or DSM): v1.0.0
- Willing to develop solution: Yes
- Has workaround: Yes
- High impact: No

**Comment by varunj90** [comment]
hey @zyl-edison-telus, can you triage this issue please. thanks!

**Comment by theetrain** [comment]
This is now fixed in [@tds/community-input-group@1.0.1](https://github.com/telus/tds-community/releases/tag/%40tds%2Fcommunity-input-group%401.0.1). Thank you for your contributions!
